packages/@o3r/styling/README.md

```diff
@@ -1,6 +1,13 @@
-# Otter styling
+<h1 align="center">Otter styling</h1>
+<p align="center">
+  <img src="../../../.attachments/otter.png" alt="Super cute Otter!" width="40%"/>
```

Comment:
do we keep this alt? :D

Reply:
Why not, it is matching the description of the picture, right?